sections/header.liquid

```diff
@@ -240,6 +244,7 @@
     {%- if section.settings.logo_position == 'top-center' or section.settings.menu == blank -%}
       <details-modal class="header__search">
         <details>
+          <div class="modal-overlay"></div>
```

Comment:
Invalid HTML; `div` not a valid child of `details`.

Reply:
I've moved the overlay inside of the `<div>`